livre/core/render/FrameInfo.cpp

```diff
@@ -23,9 +23,11 @@ namespace livre
 {
 
 FrameInfo::FrameInfo( const Frustum& frustum_,
-                      const uint32_t frameId_ )
+                      const uint32_t frameId_,
+                      const uint32_t frameCounter_ )
```

Comment:
FrameId says which Livre frame ( encoded in nodes ) you are rendering. FrameCounter is incremented by Equalizer. Smae FrameId can be rendered different counters.